src/main/java/com/google/api/codegen/py/PythonSphinxCommentFixer.java

```diff
@@ -22,6 +22,8 @@
 
   /** Returns a Sphinx-formatted comment string. */
   public static String sphinxify(String comment) {
+    // escape '$' in the comment first
+    comment = comment.replaceAll("\\$", "\\\\\\$");
```

Comment:
http://docs.oracle.com/javase/7/docs/api/java/lang/String.html#replace(java.lang.CharSequence,%20java.lang.CharSequence) This can be used?